src/main/java/org/example/Algorithms/dijkstras.java

```diff
@@ -0,0 +1,72 @@
+/*
+     Write a program to implement Dijkstra's Algorithm. 
+*/
+
+import java.util.*;
+
+class dijkstraClass{
```

Comment:
Please rename class to DijkstraClass to follow _CamelCase_ convention for class names.

Reply:
okay i'll add it right away.